src/modules/BasicTH.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from "prop-types"
+import React from "react"
+
+/**
+ * This is the basic replacement for a TH component.
+ * If you want to use your own custom react component, instead of a raw
+ * TH component, you need to make sure it calls 'props.updateCellWidth',
+ * and sets its own width off of 'props.width'
```

Comment:
where is this TH setting its width?

Reply:
the width prop goes straight through to the `th` with `...rest`

Reply:
I added width as an explicit prop to better illustrate what to do.